.github/workflows/verify_p.yml

```diff
@@ -0,0 +1,45 @@
+# This is a basic workflow to verify incoming PRs with just one single machine (openvario-57-lvds)
+name: verify_pr
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  #push:
+    #branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  Verify_PR:
+    
+    name: Verify recipes at Pull Requests
+    
+    # The type of runner that the job will run on
+    runs-on: ubuntu-20.04
+    
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+                
+      - name: Checkout sources
+        uses: actions/checkout@v2
+        
+      - name: Update submodules
+        run: git submodule update --init
+      
+      # Install necessary packages
+      - name: Install build dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y ffmpeg gawk wget git diffstat unzip texinfo gcc build-essential chrpath socat cpio python3 python3-pip python3-pexpect xz-utils debianutils iputils-ping python3-git python3-jinja2 libegl1-mesa libsdl1.2-dev pylint3 xterm python3-subunit mesa-common-dev zstd liblz4-tool
```

Comment:
What's the use of these packages? They are only needed to actually build XCSoar.

Reply:
Yes. I am also working on a different container for building complete images, not only checking recipes. I try to minimize the variants of the workflow files. 
For only checking the recipes (--dry-run) these packages are not necessary.